src/main/resources/static/api.yml

```diff
@@ -2628,6 +2628,24 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /cache/{cacheName}/clear:
```

Comment:
With my REST hat on, we should just call this `/cache/{cacheName}`, as `cacheName` is the thing we're `DELETE`ing

Reply:
Good point, have switched it